src/lib/y2storage/dialogs/guided_setup/widgets/linux_partition_actions.rb

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact Novell, Inc.
+#
+# To contact Novell about this file by physical or electronic mail, you may
+# find current contact information at www.novell.com.
+
+require "yast"
+require "y2storage"
+require "y2storage/dialogs/guided_setup/widgets/base"
+
+module Y2Storage
+  module Dialogs
+    class GuidedSetup
+      module Widgets
+        # Widget to select the actions to perform over the Linux partitions
+        class LinuxPartitionActions < Base
+          # Whether the widget should be enabled by default
+          #
+          # @return [Boolean]
+          attr_reader :enable_on_init
+
+          alias_method :enable_on_init?, :enable_on_init
+
+          # Constructor
+          #
+          # @param widget_id [String]
+          # @param settings [Y2Storage::ProposalSettings]
+          # @param enabled [Boolean]
+          def initialize(widget_id, settings, enabled: true)
+            super(widget_id, settings)
+
```

Comment:
np: why these blank lines?

Reply:
Only a matter of taste. I tend to separate lines that are not related at all.